Tests/FunctionalTests/Sources/FTRGestureTest.m

```diff
@@ -15,6 +15,7 @@
 //
 
 #import "FTRBaseIntegrationTest.h"
+#import <EarlGrey/EarlGrey.h>
```

Comment:
QQ - shouldn't this be above the FTRBaseIntegrationTest import above? Throughout?

Reply:
Hmm in Google repo it was in this order. I can manually update it though

Reply:
Discussed offline. There a lot of import ordering changes needed for Functional tests about EarlGrey.h. Will do that in another pull request.